docs/tutorial/tests.rst

```diff
@@ -469,6 +469,7 @@ should show an error message on invalid data.
         auth.login()
         response = client.post(path, data={'title': '', 'body': ''})
         assert b'Title is required.' in response.data
+	assert post['body'] == ''
```

Comment:
Not indented with rest of test. `post` is undefined.

Fixing those, this change also needs to be made in the corresponding example code, not only in the docs.